ui/src/modals/newtask.vue

```diff
@@ -276,13 +286,26 @@ export default {
                             value: resource.id,
                             text: resource.name + " - " + resource.info.name
                         };
-                    })
+                    });
                 }
                 
                 this.preferrable_resources.unshift({
                     value: null,
                     text: '(None)'
                 });
+                
```

Comment:
We should refactor this into <advancedoptions> component and use it on both newtask and appsubmit.

Reply:
and done